### DISCUSSION TITLE
How to Hide Items in Smart Collections

### DISCUSSION THREAD

**Discussion by ExtraChoccyMilk** [meisnate12/Plex-Meta-Manager#734] [Q&A] [answered]
I am probably just missing the setting/answer somewhere else, but hopefully someone can help me here.

How do I hide movies/shows that are part of smart collections. I am only using smart collections with PMM/Plex, but for some reason none of the movies are hiding. I have tried hiding them at the library level and the collection level, but they still show in the library tab no matter what I do.

**Comment by YozoraXCII** [comment, marked as answer]
It is not currently possible to hide movies that appear within Smart Collections, this is a limitation of Plex itself - the behavior will be the same if you manually create a Smart Collection within the Plex UI.

I'm not sure if this is deemed a bug in Plex or if it is by design.

**Reply by ExtraChoccyMilk** [reply]
Thanks for the answer! That was my fear :( Oh well, chalk it up to another Plex "feature" or bug. haha

**Comment by ExtraChoccyMilk** [comment]
I figured out a work around for my issue (I am sure other people have figured this out too). If you create one (and only one) normal collection, you can hide all items within that collection. So, I created a "Library" collection that is all items within that library, then hide the items within that collection. 

The script I used was:
```

  Library:
    sort_title: +!2_<<num>><<collection_name>>
    sync_mode: sync
    collection_order: alpha
  Plex Library:
    template: { name: Library, num: 1 }
    plex_all: true
    summary: Plex library sorted by title

```
